### PR TITLE
Update fw fligth mode descriptions

### PR DESCRIPTION
**Pull request by RomanBapst**

### CODE REVIEW

**Comment by RomanBapst** [comment]
@hamishwillee Let me know what you think about it. I found that e.g. in Altitude and Position mode the roll stick controls roll angle the same way as in Stabilized mode so I just added references to avoid repeating the same sentence again. However, I'm not sure how good this is in terms of user experience since the flight modes are not self-explanatory anymore.

File: en/flight_modes/README.md [hunk 1]
```diff
@@ -131,8 +131,13 @@ th {
  <td>M</td>
  <td></td>
  <td>
-  <p>RC mode where centered RP sticks level vehicle (centered sticks put vehicle into straight and level flight). The vehicle course and altitude are not maintained, and can drift due to wind.</p>
-  <p>If roll/pitch sticks are non-zero the vehicle does a coordinated turn (manual yaw input is added to rudder control input to control sideslip).</p>
+  <p>RC mode where centered RP sticks levels vehicle attitude (roll and pitch). The vehicle course and altitude are not maintained, and can drift due to wind.</p>
+<ul>
+   <li>Pitch stick controls pitch angle.</li>
+   <li>Roll stick controls roll angle. Autopilot will maintain coordinated flight.</li>
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/flight_modes/README.md]: @bkueng Are you happy with the changes in this section? I think they are clear provided there is a link to co-ordinated flight as above.

**Comment by hamishwillee** [comment]
Thanks @RomanBapst !

> @hamishwillee Let me know what you think about it. I found that e.g. in Altitude and Position mode the roll stick controls roll angle the same way as in Stabilized mode so I just added references to avoid repeating the same sentence again. However, I'm not sure how good this is in terms of user experience since the flight modes are not self-explanatory anymore.

Background. The current table is ordered from easiest to hardest. I expanded out the text so that users don't have to jump around between sections to understand how things work AND because then I could reproduce this text more or less unchanged in the mode-specific detail pages. That is not to say that this isn't something we can't change, but I want you to understand my reasoning before we do so.

Generally I'm OK with the idea of reducing what users have to read. In particular, if user understands one mode then referencing another speeds up the learning process. 

The problem with this is order of introduction - the position mode (encountered first) related to altitude mode, which has not yet been introduced. So the user does not benefit from previous learning. This might be OK if altitude was first. I don't really want to reorder because the easiest to hardest ordering makes sense to me (?).

So suggest we either:
-  add the expanded version back 
- Turn this around so that Position mode explains the behaviour and Altitude mode says "like position mode".
- Have both - so for position: "Pitch stick controls altitude (like Altitude mode)"

I lean towards the third option.

**Comment by hamishwillee** [comment]
Note to self - update the sub pages as necessary following these changes.

**Comment by RomanBapst** [comment]
Let me make another pass

**Comment by hamishwillee** [comment]
OK. Let me know when you're ready! Thanks.

**Comment by RomanBapst** [comment]
@hamishwillee I think it's very understandable! I think we can merge it.

**Comment by hamishwillee** [comment]
@RomanBapst Will do thanks! 

We are missing some information - the relevant parameters in the sub pages for Fixed wing modes - e.g. see https://docs.px4.io/en/flight_modes/position_fw.html#parameters ... you can see the similar version for MC has information about the r[elevant configuration params](https://docs.px4.io/en/flight_modes/position_mc.html#parameters). Can you look at those too and make suggestions about the params that are most relevant?

**Pull request merged**